tools/pytorch2onnx.py

```diff
@@ -82,9 +75,122 @@ def pytorch2onnx(model,
         bbox_results = bbox2result(det_bboxes, det_labels, 1)
         onnx_results = bbox_results[0]
         assert np.allclose(
-            pytorch_result[0][:, 4], onnx_results[:, 4]
-        ), 'The outputs are different between Pytorch and ONNX'
-        print('The numerical values are same between Pytorch and ONNX')
+            pytorch_result[0][0][0][:4], onnx_results[0]
+            [:4]), 'The outputs are different between Pytorch and ONNX'
+        print('The numerical values are the same between Pytorch and ONNX')
+
+
+def generate_inputs_and_wrap_model(config_path, checkpoint_path, input_config):
```

Comment:
In our design, each framework (MMDet, MMSeg...) should provide this function and AutoSDK will import the same function (at least the same name) while dealing with different framework. So I highly suggest to put this function into the framework, not just defined it ```tools/pytorch2onnx.py```

Reply:
Thanks and I have added the `generate_inputs_and_wrap_model` and its required helper function in `mmdet.core.export`.